MvvmCross/Core/Core/Platform/LogProviders/ConsoleLogProvider.cs

```diff
@@ -178,7 +178,8 @@ private static class ConsoleColorValues
 
             static ConsoleColorValues()
             {
-                Type = Type.GetType("System.ConsoleColor");
+                Type = Type.GetType("System.ConsoleColor") ?? Type.GetType("System.ConsoleColor, System.Console");
```

Comment:
@willsb what do you think of this?

Reply:
I was actually thinking we can drop the initial GetType and just use the assembly qualified call, but will that work across all platforms?

Reply:
When I started working on this I had issues with  Type.GetType("System.ConsoleColor") on runtime, always returning null. So the workarround with Type.GetType("System.ConsoleColor, System.Console"); worked for me.